code/espurna/sensors/DHTSensor.h

```diff
@@ -204,7 +204,7 @@ class DHTSensor : public BaseSensor {
                 _humidity = dhtData[0];
             } else if (_type == DHT_CHIP_DHT12) {
                 _humidity = dhtData[0];
-				_humidity += dhtData[1] * 0.1;
+				        _humidity += dhtData[1] * 0.1;
```

Comment:
You may want to revert this file the changes in here are useless

Reply:
Yes, please, revert changes to the DHTSensor.h file.

Reply:
Reverted!